hw/top_earlgrey/data/ip/chip_clkmgr_testplan.hjson

```diff
@@ -0,0 +1,369 @@
+              "chip_sw_clkmgr_external_clk_src_for_sw_slow_test_unlocked0",
+              "chip_sw_clkmgr_external_clk_src_for_sw_fast_dev",
+              "chip_sw_clkmgr_external_clk_src_for_sw_slow_dev",
+              "chip_sw_clkmgr_external_clk_src_for_sw_fast_rma",
+              "chip_sw_clkmgr_external_clk_src_for_sw_slow_rma",
+              "manuf_cp_unlock_raw",
+	      "manuf_cp_yield_test"]
+      bazel: [
+        "//sw/device/tests:ast_clk_outs_test_fpga_cw310_test_rom",
+	"//sw/device/tests:clkmgr_external_clk_src_for_sw_fast_test_fpga_cw310_test_rom",
+	"//sw/device/tests:clkmgr_external_clk_src_for_sw_slow_test_fpga_cw310_test_rom",
+      ]
+    }
+    {
+      name: chip_sw_clkmgr_external_clk_src_for_lc
+      desc: '''Verify the clkmgr requests ext clk src during certain LC states.
+
+            On POR lc asserts lc_clk_byp_req on some LC states, and de-asserts
+            it when lc_program completes. This also triggers divided clocks to step down. It may be
+            best to verify this via SVA, unless we implement clock cycle counters.
+	    SiVal: There is no way to test clock frequencies before clocks are calibrated.
+	    So the only way to actually test this is by the success of OTP updates for calibration.
+	    This can be tested after the fact by clock measurements of the internal, calibrated clocks.
+            '''
+      stage: V2
+      si_stage: SV1
+      lc_states: [
+        "RAW",
+	"TEST_LOCKED0",
+      ]
+      tests: [
+        "chip_sw_clkmgr_external_clk_src_for_lc",
+	"manuf_cp_unlock_raw",
+      ]
+    }
+    {
+      name: chip_sw_clkmgr_external_clk_src_for_sw
+      desc: '''Verify SW causes the clkmgr requests ext clk src during certain LC states.
+
+            In DEV, TEST_UNLOCKED, and RMA lc states the external clock is enabled in response to
+            `extclk_ctrl.sel` CSR writes. In addition `extclk_ctrl.hi_speed_sel` CSR causes the
+            divided clocks to step down. Verify this via SVA bound to clkmgr, and clock cycle
+            counters.
+
+            Disable external clock source and verify the AST reliably falls back to the internal
+            clock. Ensure the chip operates normally.
+            X-ref with chip_sw_uart_tx_rx_alt_clk_freq, which needs to deal with this as well.
+	    SiVal: The only necessary test is that the internal clocks are running
+	    with the right frequency after calibration.
+            '''
+      stage: V2
+      si_stage: None
+      lc_states: [
+        "TEST_UNLOCKED0",
+        "DEV",
+        "RMA",
+      ]
+      tests: ["chip_sw_clkmgr_external_clk_src_for_sw_fast_test_unlocked0",
+              "chip_sw_clkmgr_external_clk_src_for_sw_slow_test_unlocked0",
+              "chip_sw_clkmgr_external_clk_src_for_sw_fast_dev",
+              "chip_sw_clkmgr_external_clk_src_for_sw_slow_dev",
+              "chip_sw_clkmgr_external_clk_src_for_sw_fast_rma",
+              "chip_sw_clkmgr_external_clk_src_for_sw_slow_rma"]
+    }
```

Comment:
I believe very SiVal test case should have a `bazel: []` field, like here: https://cs.opensource.google/opentitan/opentitan/+/master:hw/top_earlgrey/data/ip/chip_kmac_testplan.hjson;drc=8ef4c5525d76b42009f75c6f153759e0766c4487;l=113

This will be similar to the `tests: []` field, except instead of being picked up by dvsim, it will be picked up by Bazel / maybe some future scripting tooling.

Reply:
Yes, but only for sival tests. There are tests that are only suitable for simulation, not sival. Somehow we need to mark such tests. I have being marking them as `si_stage: None` for my tracking.

Remember this testplan hjson files are also meant for sim_dv tests, not just sival.

Reply:
ah I see, the `si_state: None` just threw me off. disregard my comment then

Reply:
I added empty bazel lists anyway.